examples/00_getting_started.py

```diff
@@ -0,0 +1,220 @@
+"""
+
+# %%
+# Downloading example datasets
+# ----------------------------
+#
+# The :obj:`~skrub.datasets` module allows us to download tabular datasets and
+# demonstrate ``skrub``'s features.
+
+# %%
+from skrub.datasets import fetch_employee_salaries
+
+dataset = fetch_employee_salaries()
+employees_df, salaries = dataset.X, dataset.y
+
+# %%
+# Explore all the available datasets in :ref:`downloading_a_dataset_ref`.
+
+
+# %%
+# Generating an interactive report for a dataframe
+# -------------------------------------------------
+#
+# To quickly get an overview of a dataframe's contents, use the
+# :class:`~skrub.TableReport`.
+
+# %%
+from skrub import TableReport
+
+TableReport(employees_df)
+
+# %%
+# You can use the interactive display above to explore the dataset visually.
+
+# %%
+# Easily building a strong baseline for tabular machine learning
+# --------------------------------------------------------------
+#
+# The goal of ``skrub`` is to ease tabular data preparation for machine learning.
+# The :func:`~skrub.tabular_learner` function provides an easy way to build a simple
+# but reliable machine-learning model, working well on most tabular data.
+
+
+# %%
+from sklearn.model_selection import cross_validate
+
+from skrub import tabular_learner
+
+model = tabular_learner("regressor")
+results = cross_validate(model, employees_df, salaries)
+results["test_score"]
+
+# %%
+# To handle rich tabular data and feed it to a machine-learning model, the
+# pipeline returned by :func:`~skrub.tabular_learner` preprocesses and encodes
+# strings, categories and dates using the :class:`~skrub.TableVectorizer`.
+# See its documentation or :ref:`sphx_glr_auto_examples_01_encodings.py` for
+# more details. An overview of the chosen defaults is available in
+# :ref:`end_to_end_pipeline`.
+
+
+# %%
+# Assembling data
+# ---------------
+#
+# ``Skrub`` allows imperfect assembly of data, such as joining dataframes
+# on columns that contain typos. ``Skrub``'s joiners have ``fit`` and
+# ``transform`` methods, storing information about the data across calls.
+#
+# The :class:`~skrub.Joiner` allows fuzzy-joining multiple tables, each row of
+# a main table will be augmented with values from the best match in the auxiliary table.
+# You can control how distant fuzzy-matches are allowed to be with the
+# ``max_dist`` parameter.
+
+# %%
+# In the following, we add information about countries to a table containing
+# airports and the cities they are in:
+
+# %%
+import pandas as pd
+
+from skrub import Joiner
+
+airports = pd.DataFrame(
+    {
+        "airportId": [1, 2],
+        "airportName": ["Charles de Gaulle", "Aeroporto Leonardo da Vinci"],
+        "City": ["Paris", "Roma"],
+    }
+)
+# notice the "Rome" instead of "Roma"
+capitals = pd.DataFrame(
+    {"City": ["Berlin", "Paris", "Rome"], "Country": ["Germany", "France", "Italy"]}
+)
+joiner = Joiner(
+    capitals,
+    key="City",
+    suffix="_aux",
```

Comment:
maybe if in the "capitals" table you name it 'Capital' instead of 'City' we can avoid setting the `suffix`. it is documented elsewhere so I think we can skip it in the "getting started page"